app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:comments_count)
+  end
```

Comment:
- [OPTIONAL] As mentioned above you can use `after_save :update_comments_counter`(https://github.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/issues/4), after_save :update_comments_counter

Reply:
thank you @EricMbouwe for the suggestions, I will look into the article